### PR TITLE
Add support for Korean language

### PR DESCRIPTION
**Pull request by wangmengyan95**

### CODE REVIEW

File: ParseUI-Login/src/main/res/values-ko/strings.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<resources>
+
+    <string name="com_parse_ui_logo_content_description">Logo</string>
```
Comment (grantland) [inline ParseUI-Login/src/main/res/values-ko/strings.xml]: We don't need this if it's not translated as it'll pull the default from `strings.xml`

Comment (wangmengyan95) [inline ParseUI-Login/src/main/res/values-ko/strings.xml]: Good catch:) I delete a bunch of the duplicated strings but miss this one.

**Comment by wangmengyan95** [comment]
I ask the developer to double check it and he says it is OK.

**Comment by grantland** [comment]
LGTM

**Pull request merged**